src/main/java/com/tg5/controller/AccountsController.java

```diff
@@ -1,12 +1,41 @@
 package com.tg5.controller;
 
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
+import com.tg5.domain.Account;
+import com.tg5.repository.AccountRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
 
 @RestController
 @RequestMapping("/accounts")
 public class AccountsController {
```

Comment:
You have to extend BaseReadWriteController<AccountPayload, Account, Long> and you'll get all the necessary functionality. Try and see if it satisfies your requirements.

Reply:
okay